### PR TITLE
Another attempt to fix the broken mean test.

### PR DESCRIPTION
**Pull request by ejj**

### CODE REVIEW

File: quilt-tester/tests/30-mean/check_mean.go [hunk 1]
```diff
@@ -64,18 +67,23 @@ func httpGetTest(machines []db.Machine, containers []db.Container) {
...
+			} else {
+				log.Error(resp)
+				failed = true
 			}
-
-			log.Info(resp)
 		}
 	}
+
+	return !failed
```
Comment (secant) [inline quilt-tester/tests/30-mean/check_mean.go]: I think this should just be `failed`

File: specs/mongo/scripts/run.go [hunk 2]
```diff
@@ -7,6 +7,8 @@ import (
 	"os/exec"
```
Comment (sirspinach) [inline specs/mongo/scripts/run.go]: In the commit message: "dieing" -> "dying"

File: quilt-tester/tests/30-mean/check_mean.go (hunk 1 above)
Comment (ejj) [inline quilt-tester/tests/30-mean/check_mean.go]: No, I think it returns true if passed, false if failed.  So !failed => true if it passed.

Comment (secant) [inline quilt-tester/tests/30-mean/check_mean.go]: yeah my bad, got confused

**Pull request merged**